### DISCUSSION TITLE
Contribute to pimcore

### DISCUSSION THREAD

**Discussion by SamyMP** [pimcore/pimcore#11088] [General]
Hi !

I would like to contribute to Pimcore, add new features, correct bugs, etc... But I don't know how to install the projet. I read the Contributing.MD but I coudn't find any specific instruction on how to get it up and running, ready for developpement. Maybe a Docker could help ?

Anyway, could you please provide precise instructions on how to install pimcore to contrubute ?

Thanks !

**Comment by passioneight** [comment]
Maybe [Pimcore's documentation](https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html) can help you get started.

Also, have a look at the `Docker` section of [this repo](https://github.com/pimcore/skeleton).
